providers/implementations/kem/rsa_kem.c

```diff
@@ -235,6 +246,9 @@ static int rsasve_gen_rand_bytes(RSA *rsa_pub,
     BN_CTX *bnctx;
     BIGNUM *z, *nminus3;
 
+    if (!ossl_prov_is_running())
```

Comment:
This check should be done in `rsakem_generate()` I think.
That's the only caller of this function (via `rsasve_generate()`) and it matches the check added in `rsasve_recover()`.